pkg/endpoint/endpoint.go

```diff
@@ -500,6 +500,15 @@ func (e *Endpoint) GetModelRLocked() *models.Endpoint {
 		},
 		Controllers: e.controllers.GetStatusModel(),
 	}
+
+	// Sort these slices since they come out in random orders. This allows
+	// reflect.DeepEqual to succeed.
+	sort.StringSlice(mdl.Labels.Custom).Sort()
+	sort.StringSlice(mdl.Labels.Disabled).Sort()
+	sort.StringSlice(mdl.Labels.OrchestrationIdentity).Sort()
+	sort.StringSlice(mdl.Labels.OrchestrationInfo).Sort()
+	sort.Slice(mdl.Controllers, func(i, j int) bool { return mdl.Controllers[i].Name < mdl.Controllers[i].Name })
```

Comment:
this will always return `false` :smile: (one is `i`, then the other one should be `j`)

Reply:
oops, yes... I mean, it kinda worked ;)